bin/citrea/tests/bitcoin_e2e/batch_prover_test.rs

```diff
@@ -270,8 +270,8 @@ impl TestCase for SkipPreprovenCommitmentsTest {
             .into_iter()
             .map(|response| SequencerCommitment {
                 merkle_root: response.merkle_root,
-                l2_start_block_number: response.l2_start_block_number,
-                l2_end_block_number: response.l2_end_block_number,
+                l2_start_block_number: response.l2_start_block_number.try_into().expect("U64 to u64 must succeed"),
+                l2_end_block_number: response.l2_end_block_number.try_into().expect("U64 to u64 must succeed"),
```

Comment:
try_into().expect() is no better than .to(). But really longer. Try to revert commits with git revert